browser/brave_ads/brave_ads_host.cc

```diff
@@ -30,12 +33,33 @@ constexpr char kAdsEnableRelativeUrl[] = "request_ads_enabled_panel.html";
 
 }  // namespace
 
-BraveAdsHost::BraveAdsHost(Profile* profile) : profile_(profile) {
+BraveAdsHost::BraveAdsHost(
+    Profile* profile,
+    content::GlobalRenderFrameHostId render_frame_host_id)
+    : profile_(profile), render_frame_host_id_(render_frame_host_id) {
   DCHECK(profile_);
+  DCHECK(render_frame_host_id_);
```

Comment:
not needed

Reply:
`GlobalRenderFrameHostId` has a operator bool() which checks if GlobalRenderFrameHostId was initialized